backend/src/apiserver/common/config.go

```diff
@@ -48,8 +48,11 @@ const (
 	TerminateStatus                         string = "TERMINATE_STATUS"
 	MoveResultsImage                        string = "MOVERESULTS_IMAGE"
 	Path4InternalResults                    string = "PATH_FOR_INTERNAL_RESULTS"
+	ObjectStoreCredentialsSecret            string = "OBJECTSTORECONFIG_CREDENTIALSSECRET"
+	ObjectStoreCredentialsAccessKeyKey      string = "OBJECTSTORECONFIG_CREDENTIALSACCESSKEYKEY"
+	ObjectStoreCredentialsSecretKeyKey      string = "OBJECTSTORECONFIG_CREDENTIALSSECRETKEYKEY"
 	ObjectStoreAccessKey                    string = "OBJECTSTORECONFIG_ACCESSKEY"
-	ObjectStoreSecretKey                    string = "OBJECTSTORECONFIG_SECRETKEY"
+	ObjectStoreSecretKey                    string = "OBJECTSTORECONFIG_SECRETACCESSKEY"
```

Comment:
@rimolive I remember you introduced this new variable. In KFP-Tekton I don't think we are using this variable in open source. Are you okay with this change?

Reply:
FWIW I believe this was just a typo that I've corrected here, vanilla KFP has instances of what I've updated it to (`OBJECTSTORECONFIG_SECRETACCESSKEY`) in their manifests, but as @Tomcli noted it doesn't appear to be used anywhere here in KFP-Tekton so it never broke anything.